src/LibraryManager.Contracts/ILibraryInstallationResult.cs

```diff
@@ -23,6 +23,13 @@ public interface ILibraryInstallationResult
         /// </remarks>
         bool Success { get; }
 
+        /// <summary>
+        /// <code>True</code> if the library is up to date; otherwise <code>False</code>.
+        /// </summary>
+        /// <remarks>
+        /// </remarks>
+        bool UpToDate { get; }
```

Comment:
New status for libraries that are up to date with the one on cache

Reply:
If the library files are in sync with the one in the cache we want to log message saying that library is up to date. Just added this new property to flag in the ILibraryInstallationResult that nothing was changed for that specific library.

Reply:
We should probably rename this type to ILibraryOperationResult since we use that for all types of operations and not only Install.